### ISSUE TITLE
npm install  npm start doesn't work

### ISSUE THREAD

**Issue by mf093087** [dgg32/doctorai_eli5#6]
Hi, I want to use your chatbot. But when I ran npm install and npm start, I met this error. Do I do something wrong?
- npm install
![image](https://user-images.githubusercontent.com/67538561/190827787-606b98f3-d47e-4d69-b4a3-118d51add012.png)

- npm start 
![image](https://user-images.githubusercontent.com/67538561/190827818-924831d7-b0b4-4e07-a457-59e8753b39e3.png)

**Comment by dgg32** [comment]
Hi @mf093087 . I googled a bit, it seems it is an npm issue:

https://stackoverflow.com/questions/71582397/eresolve-unable-to-resolve-dependency-tree-while-installing-a-pacakge

https://stackoverflow.com/questions/65549858/eresolve-unable-to-resolve-dependency-tree-when-installing-npm-react-facebook

Could you please try these. Thanks.

**Comment by mf093087** [comment]
Hi I tried yarn install. then it works.

However, I see nothing on the website, is it blocked?
